docs/intro/tutorial07.txt

```diff
@@ -418,5 +418,42 @@ uses a template variable called ``app_list``. That variable contains every
 installed Django app. Instead of using that, you can hard-code links to
 object-specific admin pages in whatever way you think is best.
 
+How to completely delete an app from the project
```

Comment:
Tutorial is not the right place for How-to's, please move it to the [How-to guides](https://docs.djangoproject.com/en/stable/howto/).

Reply:
okay, is there a way to see how it would look on a website, I only see the code and do not really know what it would look like? It would really help to make it properly.

Reply:
Yes! https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#getting-started-with-sphinx

Reply:
okay, thanks, i will make best possible